filebeat/module/nginx/access/_meta/fields.yml

```diff
@@ -17,6 +17,9 @@
     - name: remote_ip
       type: alias
       path: source.ip
+    - name: origin
+      type: array
+      path: nginx.access.origin
```

Comment:
We should use [`source.address`](https://github.com/elastic/ecs#-source-fields) also when the source is a unix socket, so no need to add a field for this.

If you would need to add this field, it'd be:
```
- name: origin
  type: keyword
  description: >
     Some description
```

`path` is intended to be used with aliases, and `migration` for migrated fields.

Reply:
For the use of `keyword` instead of `array`, take a look to https://www.elastic.co/guide/en/elasticsearch/reference/6.6/array.html

Reply:
Those fields have been moved to ECS so no need to update fields here